bundles/org.openhab.binding.asuswrt/src/main/resources/OH-INF/thing/router.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="asuswrt"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Router Thing Type -->
+	<bridge-type id="router">
+		<label>Asuswrt Router</label>
+		<description>Router with asuswrt or asuswrt-merlin OS</description>
+		<channel-groups>
+			<channel-group id="sys-info" typeId="sys-info-group"></channel-group>
+			<channel-group id="client-list" typeId="client-list-group"></channel-group>
+		</channel-groups>
+
+		<config-description>
+			<parameter name="hostname" type="text" required="true">
+				<context>network-address</context>
+				<label>Hostname</label>
+				<description>Hostname or IP-Address of the device</description>
```

Comment:
AFAIK hypens are used like this in German (and related languages) but not in English. See [IP address](https://en.wikipedia.org/wiki/IP_address) on Wikipedia. Can you also check/fix the other hypens? E.g. I see many MAC-Address occurrences in this PR but it also seems to be spelled like [MAC address](https://en.wikipedia.org/wiki/MAC_address) on Wikipedia.

```suggestion
				<description>Hostname or IP address of the device</description>
```